documentation/modules/cruise-control/con-optimization-proposals.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// assembly-cruise-control-concepts.adoc
+
+[id='con-optimization-proposals-{context}']
+
+= Optimization proposals overview
+
+An _optimization proposal_ is a summary of proposed changes that would produce a more balanced Kafka cluster, with partition workloads distributed more evenly among the brokers. 
+Each optimization proposal is based on the set of xref:con-optimization-goals-{context}[optimization goals] that were used to generate it.
+
+Optimization proposals always satisfy all the configured hard goals, although not necessarily all the soft goals. Hard goals are ignored if `skipHardGoalCheck: true` is specified in the `KafkaRebalance` custom resource.
+
+An optimization proposal contains the following information, in JSON, about the pending cluster rebalance:
```

Comment:
So an optimization proposal does contain the information in the bullet points below but we don't currently show this information to a user. This requires the `--verbose` flag which we currently don't support. We may add this in future but there is currently no way to show this information to the user.

Reply:
You could say "An optimisation proposal consists of the following information........however a summary of the changes is displayed in the KafkaRebalance CR status."

Reply:
Removed the bullet points.

Reply:
Please can you check this again, @tomncooper ?

Reply:
This section has changes so there is nothing to check. You can resolve this.